stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/Decorator.kt

```diff
@@ -15,10 +16,16 @@ internal abstract class BaseDecorator : Decorator {
         when (viewHolder) {
             is MessageDeletedViewHolder -> decorateDeletedMessage(viewHolder, data as MessageListItem.MessageItem)
             is MessagePlainTextViewHolder -> decoratePlainTextMessage(viewHolder, data as MessageListItem.MessageItem)
+            is OnlyMediaAttachmentsViewHolder -> decorateOnlyMediaAttachmentsMessage(viewHolder, data as MessageListItem.MessageItem)
             else -> Unit
         }.exhaustive
     }
 
+    abstract fun decorateOnlyMediaAttachmentsMessage(
```

Comment:
Perhaps these methods here could be `open`, and empty by default? Might get rid of a lot of empty methods in the concrete `Decorator` classes in the long run.

Reply:
Good point